medusa/providers/nzb/newznab.py

```diff
@@ -44,7 +45,7 @@ class NewznabProvider(NZBProvider):
     Tested with: newznab, nzedb, spotweb, torznab
     """
 
-    def __init__(self, name, url='', api_key='0', cat_ids='5030,5040', default=False, search_mode='eponly',
+    def __init__(self, name, url='', api_key='0', cat_ids=('5030', '5040'), default=False, search_mode='eponly',
```

Comment:
I would make `cat_ids` optional at this point since they are being assigned in the constructor anyway.

Reply:
I would make cat_ids optional at this point since they are being assigned in the constructor anyway.